tests/mentionui.js

```diff
@@ -897,6 +897,49 @@ describe( 'MentionUI', () => {
 							expectChildViewsIsOnState( [ true, false, false, false, false ] );
 						} );
 				} );
+
+				it( 'should not cycle when only one item in the list', () => {
+					setData( model, '<paragraph>foo []</paragraph>' );
+					const keyDownEvtData = {
+						keyCode: keyCodes.arrowdown,
+						preventDefault: sinon.spy(),
+						stopPropagation: sinon.spy()
+					};
+
+					const keyUpEvtData = {
+						keyCode: keyCodes.arrowdown,
+						preventDefault: sinon.spy(),
+						stopPropagation: sinon.spy()
+					};
+
+					model.change( writer => {
+						writer.insertText( '@', doc.selection.getFirstPosition() );
+					} );
+
+					return waitForDebounce()
+						.then( () => {
+							expectChildViewsIsOnState( [ true, false, false, false, false ] );
```

Comment:
This unit test should be simplified. It should only focus on testing what's in the description, so case with a single item and pressing down/up arrow keys. No need to make test step to test all the items.

So the line https://github.com/ckeditor/ckeditor5-mention/blob/fe2b5c0/tests/mentionui.js#L916 should be more like:

```js
writer.insertText( '@T', doc.selection.getFirstPosition() );
```

And lines https://github.com/ckeditor/ckeditor5-mention/blob/fe2b5c0/tests/mentionui.js#L920-L930 should be removed.

Reply:
Sure, looks like I've copy-pasted too much :D